### DISCUSSION TITLE
File size increase?

### DISCUSSION THREAD

**Discussion by Bketting** [google/model-viewer#2667] [Q&A]
Since some weeks the export .glb has increased tremendously. Uploading the attached files (.glb without textures, and RGB, MRO, normal maps) results in a 28MB .glb export including textures. The individual files are 2-3MB in total. What is happening?

![PBK_Color_2K](https://user-images.githubusercontent.com/45426768/128560949-09808852-ec40-4127-a1cb-11af9042ac47.jpg)
![PBK_MRO_2K](https://user-images.githubusercontent.com/45426768/128560957-ffac5290-4277-4b26-a6a6-39cbd9789f86.jpg)
![PBK_Normal_2K](https://user-images.githubusercontent.com/45426768/128560959-55714c4b-65f7-41f2-9429-3e05a53269f2.jpg)
https://www.dropbox.com/t/d5TBLJrlisMYGzYG

**Comment by elalish** [comment]
I have a repro; the problem is your JPEGs are being turned into PNGs. We recently switched exporters, but I was unaware of this difference. Thanks, we'll get this fixed!
